x-pack/test/saved_object_api_integration/spaces_only/apis/max_size.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+
+import { FtrProviderContext } from '../../common/ftr_provider_context';
+
+export const MAPPING_BUFFER_LIMIT = 800;
+
+export default ({ getService }: FtrProviderContext) => {
+  const es = getService('es');
+
+  describe('saved object mappings', () => {
+    it(`should not exceed ${MAPPING_BUFFER_LIMIT}, if you see this test failing you should re-evaluate the number of saved object mappings`, async () => {
```

Comment:
Maybe we could either in the test name or at least in a comment in code, reference the original issue that contains a lot of hints (https://github.com/elastic/kibana/issues/43673)? Maybe in the test name `(see elastic/kibana#43673)`?

Reply:
Yea sure!